src/NServiceBus.Core/Headers.cs

```diff
@@ -66,7 +66,7 @@ public static partial class Headers
         public const string NServiceBusVersion = "NServiceBus.Version";
 
         /// <summary>
-        /// Used in a header when doing a callback (bus.return).
+        /// Used in a header when doing a callback (session.return).
```

Comment:
Hmm... Since callbacks are no longer in core. Should this go away? We can treat this also outside this PR